README.md

```diff
@@ -1,424 +1,204 @@
-```bash
-btcli root senate_vote --proposal=[PROPOSAL_HASH]
-```
+## Upgrade
 
-#### Registering for Senate
-```bash
-btcli root register
-```
+If you already installed Bittensor SDK, make sure you upgrade to the latest version. Run the below command:
 
-#### Leaving Senate
 ```bash
-btcli root undelegate
+python3 -m pip install --upgrade bittensor
 ```
 
-#### Voting in Senate
-```bash
-btcli root senate_vote --proposal=[PROPOSAL_HASH]
-```
-
-#### Miscellaneous Commands
-```bash
-btcli legacy update
-btcli legacy faucet
-```
+---
 
-#### Managing Subnets
-```bash
-btcli subnets list
-btcli subnets create
-```
+## Install on macOS and Linux
 
-#### Managing Wallets
-```bash
-btcli wallet list
-btcli wallet transfer
-```
+You can install Bittensor SDK on your local machine in either of the following ways. **Make sure you verify your installation after you install**:
+- [Install using a Bash command](#install-using-a-bash-command).
+- [Install using `pip3 install`](#install-using-pip3-install)
+- [Install from source](#install-from-source)
 
-### Note
+### Install using a Bash command
 
-Please replace the subcommands and arguments as necessary to suit your needs, and always refer to `btcli --help` or `btcli <command> --help` for the most up-to-date and accurate information.
+This is the most straightforward method. It is recommended for a beginner as it will pre-install requirements like Python, if they are not already present on your machine. Copy and paste the following `bash` command into your terminal:
 
-For example:
 ```bash
-btcli subnets --help
-
-usage: btcli <command> <command args> subnets [-h] {list,metagraph,lock_cost,create,register,pow_register,hyperparameters} ...
-
-positional arguments:
-  {list,metagraph,lock_cost,create,register,pow_register,hyperparameters}
-                        Commands for managing and viewing subnetworks.
-    list                List all subnets on the network.
-    metagraph           View a subnet metagraph information.
-    lock_cost           Return the lock cost to register a subnet.
-    create              Create a new bittensor subnetwork on this chain.
-    register            Register a wallet to a network.
-    pow_register        Register a wallet to a network using PoW.
-    hyperparameters     View subnet hyperparameters.
-
-options:
-  -h, --help            show this help message and exit
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/opentensor/bittensor/master/scripts/install.sh)"
 ```
 
-### Post-Installation Steps
+**For Ubuntu-Linux users**
+If you are using Ubuntu-Linux, the script will prompt for `sudo` access to install all required apt-get packages.
 
-To enable autocompletion for Bittensor CLI, run the following commands:
+### Install using `pip3 install`
 
 ```bash
-btcli --print-completion bash >> ~/.bashrc  # For Bash
-btcli --print-completion zsh >> ~/.zshrc    # For Zsh
-source ~/.bashrc  # Reload Bash configuration to take effect
+pip3 install bittensor
```

Comment:
I would suggest pushing the user to install bittensor in the local python environment created for this purpose.
Kind of 
```bash
python3 -m venv venv
source venv/bin/activate
pip install bittensor
```

Reply:
I used "myenv" for my environment name, just to avoid confusion between "venv" command option.

Reply:
I don't mind. I just want to add that `python -m venv venv` is something like a standard in this process that I have encountered most along the way.
But if we want to use the name other than the default one, I would prefer to use something more explicit. For example: `python3 -m venv bt_venv` or `python3 -m venv bittensor_venv`.
But again, it's up to you.